src/app.jsx

```diff
@@ -320,9 +331,14 @@ class Application extends React.Component {
         /* The following events do not need to trigger any state updates */
         case 'attach':
         case 'exec':
+        case 'exec_died':
         case 'export':
         case 'import':
         case 'init':
+        case 'mount':
+        case 'restart':
+        case 'sync':
+        case 'unmount':
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1324-20230713-103552-f61ee821-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)